src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,37 @@
+.stopwatch {
+  position: relative;
+  @include circle($clock-size);
+  border: thick double $stopwatch-border-color;
```

Comment:
You shouldn't name your color variables with word color, this is not descriptive, also don't tie the variable to block specifically, rather use generic names describing the actual color like for example `dark-grey`